javascript/javascript1/week4/Homework/highest_and_lowest.js

```diff
@@ -0,0 +1,18 @@
+console.log("===========Highest and Lowest===========");
+
+function highAndLow(numbers) {
+  numbers = numbers.split(" ");
+  for (let i = 0; i < numbers.length; i++) {
+    numbers.sort((a, b) => b - a);
```

Comment:
`a` and `b` are both strings. In JavaScript, when you perform a mathematical operation (such as `a - b`), the language automatically converts the strings to numbers, allowing your code to work. However, if you attempt to run `highAndLow("1 2 3 car 5");`, it might cause an error. Therefore, I recommend to explicitly convert the strings to numbers before doing any mathematical operations.